src/pages/api/periodValidity.ts

```diff
@@ -94,11 +98,15 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
             const period: DecisionData = {
                 operatorName: operator,
                 type: periodTypeName,
-                productName,
-                productPrice,
-                daysValid,
-                expiryRules: periodValid,
                 nocCode,
+                product: [
```

Comment:
We should keep this as products to match the multiple products structure so the netex convertor can interpret it in the same way